src/webviews/components/issue/JiraIssuePage.tsx

```diff
@@ -72,6 +72,17 @@ export default class JiraIssuePage extends AbstractIssueEditorPage<Emit, Accept,
         this.state = emptyState;
     }
 
+    // TODO: proper error handling in webviews :'(
+    // This is a temporary workaround to hopefully troubleshoot
+    // https://github.com/atlassian/atlascode/issues/46
+    getInputMarkup(field: FieldUI, editmode?: boolean, context?: String) {
+        if (!field) {
+            console.error(`Field error - no field when trying to render ${context}`);
```

Comment:
Not sure this should be error given all the versions of JIRAs out there. Perhaps a warn instead?

Reply:
Replaced `console.error` with `console.warn`:

![image](https://github.com/user-attachments/assets/3f6babd9-89f8-4bcd-8f90-7119e3bddac5)